schemas/error-schema.json

```diff
@@ -58,11 +65,18 @@
               "type": "object",
               "description": "Resource properties of the error",
               "required": [
+                "id",
                 "method",
                 "status_code",
                 "url"
               ],
               "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "UUID of the resource",
+                  "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$",
+                  "readOnly": true
+                },
```

Comment:
as discussed together, since resource errors should be removed soon, it can not worth the trouble.
And the goal of the task is more to identify each events rather than add extra links between events.